apps/wing-console/console/astrotest.js

```diff
@@ -0,0 +1,7 @@
+export const bucket = (function(env) {
+  let handle = process.env[env];
+  if (!handle) {
+    throw new Error("Missing environment variable: " + env);
+  }
+  return $simulator.findInstance(handle);
```

Comment:
Can we create a new instance of the bucket client instead, like if the simulator didn't exist?